### ISSUE TITLE
Alienware m16 R1 - keyboard ABNT2 - Brasilian layout

### ISSUE THREAD

**Issue by amarildolsouza** [T-Troll/alienfx-tools#455]
**Describe the bug**

Device:  VID_0D62/PID_D2B0, 90 lights, APIv5

Key "\" + "|" is not mapped.
Key "]" + "}" is not mapped.
Key "Ç" is equal to ";".
Key "/" is  equal to "Right Control".

Photos of keyboard:
![20240921_135239](https://github.com/user-attachments/assets/c32dbc22-3eb6-4ffe-bf55-6989f001ed37)
![20240921_135302](https://github.com/user-attachments/assets/aecaa746-b438-4ea9-9e67-a02a37f6bd2d)

Screenshot of AlienFX Control
![Screenshot 2024-09-21 135540](https://github.com/user-attachments/assets/d64c81e0-63c4-4ce4-a9e1-2f78a8d40a99)

Thanks for any help.

Have a great day.

**To Reproduce**

**Expected behavior**

**Screenshots**

**System (please complete the following information):**
 - Model: [e.g. AlienWare m15R3]
 - Model details: [e.g. per-key rgb keyboard]
 - OS: [e.g. Windows 11 stable]

**Additional context**

**Comment by T-Troll** [comment]
Yes, i have English (well... not exactly) keyboard, layout is different.
Can you please edit mappings and grid, then share me your version using "Save lights" button?
I'll add it as m16R1 (Brazil).

**Comment by amarildolsouza** [comment]
> Yes, i have English (well... not exactly) keyboard, layout is different. Can you please edit mappings and grid, then share me your version using "Save lights" button? I'll add it as m16R1 (Brazil).

Yes, off course.
Amazing.

Thanks!

---

Lights and grids:
[Alienware m16R1 (Brazil).csv](https://github.com/user-attachments/files/17089821/Alienware.m16R1.Brazil.csv)

**Comment by amarildolsouza** [comment]
> Yes, i have English (well... not exactly) keyboard, layout is different. Can you please edit mappings and grid, then share me your version using "Save lights" button? I'll add it as m16R1 (Brazil).

Hello!

If you have finished the file for the ABNT2 Brazil keyboard layout, can you please make it available?

Thanks!

**Comment by T-Troll** [comment]
Ok. it will be included into next release.
